static/app/views/dashboards/widgetBuilder/utils/convertBuilderStateToWidget.ts

```diff
@@ -22,17 +23,34 @@ export function convertBuilderStateToWidget(state: WidgetBuilderState): Widget {
     defined(state.legendAlias) && state.legendAlias.length > 0 ? state.legendAlias : [];
 
   const fieldAliases = state.fields?.map(field => field.alias ?? '');
-  const aggregates =
-    (state.yAxis?.length ?? 0) > 0
-      ? state.yAxis?.map(generateFieldAsString)
-      : state.fields
-          ?.filter(field =>
-            [FieldValueKind.FUNCTION, FieldValueKind.EQUATION].includes(
-              field.kind as FieldValueKind
-            )
+  let aggregates = [];
```

Comment:
```suggestion
  let aggregates: string[];
```
nit: Seems like we don't use the default empty `[]` anyways and this also avoids using `any[]` type.